husky-communication/husky-communication-gen/src/main/java/org/husky/communication/ConvenienceCommunication.java

```diff
@@ -996,13 +1003,17 @@ private Response submit(SecurityHeaderElement security, Association association)
             }
         }
 
+		log.info("prepare submit of document");
 		boolean secure = this.affinityDomain.getRepositoryDestination().getUri().toString().contains(HTTPS_LITERAL);
-        final var endpoint = String.format(
-                "xds-iti41://%s?inInterceptors=%s&inFaultInterceptors=%s&outInterceptors=%s&outFaultInterceptors=%s&secure=%s&audit=%s&auditContext=%s",
+		Map<String, Object> properties = new HashMap<>();
+		properties.put("mtomEnabled", true);

```

Comment:
This can be removed

Reply:
Always enabled by IPF: https://github.com/oehf/ipf/blob/master/commons/ihe/xds/src/main/java/org/openehealth/ipf/commons/ihe/xds/XDS.java#L121
Not sure where does the "mtomEnabled" comes from